index.html

```diff
@@ -12,6 +12,12 @@
 
 perks:
 
+- perl: {name: Codacy, uri: 'https://www.codacy.com/'}
```

Comment:
It looks like there's a typo on this line. Can you change `perl` to `perk`?

Also, the uri value here should like to where Codacy says that it's free for OSS projects. That looks like it's https://www.codacy.com/pricing